doc/source/examples/cfd-dem/cylindrical-packed-bed/cylindrical-packed-bed.rst

```diff
@@ -0,0 +1,370 @@
+==================================
+Cylindrical Packed Bed
+==================================
+
+It is strongly recommended to visit `CFD-DEM parameters <../../../parameters/unresolved_cfd-dem/unresolved_cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters of CFD-DEM.
+
+Features
+----------------------------------
+- Solvers: ``gls_vans_3d``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+
+
+Location of the example
+------------------------
+ ``/examples/cfd-dem/cylindrical_packed_bed``
+
+
+Description of the case
```

Comment:
Maybe seperate clearly the DEM initialization and the VANS solver part? Like explain the simulation in two parts.